api/api/urls/__init__.py

```diff
@@ -20,10 +20,14 @@
 from rest_framework.routers import DefaultRouter
 
 from api.api.views import GenerateTeamsViewSet
+from api.api.views.create_class_composition import CreateClassCompositionViewSet
 
 router = DefaultRouter()
 
 router.register(r"generate", GenerateTeamsViewSet, basename="generate_teams")
+router.register(
+    r"create", CreateClassCompositionViewSet, basename="create_class_composition"
```

Comment:
endpoint name suggestion -> "simulate people"

Reply:
like it would be `simulate/people` to use this endpoint, in general called `simulate` and in the view called `people`